lib/Resolver.js

```diff
@@ -78,6 +78,19 @@ Resolver.prototype.resolve = function resolve(context, path, request, callback)
 	}, null));
 };
 
+Resolver.prototype.doResolveWithErrorLogging = function doResolveWithErrorLogging(type, request, message, callback, options) {
+	var optionsWithoutLog = {
+		stack: options.stack,
+		missing: options.missing
+	};
+
+	this.doResolve(type, request, message, createInnerCallback((err, result) => {
```

Comment:
Do we need this outer `createInnerCallback` call? I know it isn't doing much, but this code path is hit a lot.